.github/workflows/benchmark_execution_time.yml

```diff
@@ -0,0 +1,139 @@
+      run: |
+        sudo apt -y update
+        sudo apt install libsystemd-dev librust-libdbus-sys-dev libseccomp-dev
+
+    - name: Setting rust toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        override: true
+        profile: minimal
+
+    - name: Building main branch
+      run: ./build.sh --release
+
+    - name: Uploading main build to artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: main-youki
+        path: ./youki
+
+  benchmark-exec:
+    if: contains( github.event.pull_request.labels.*.name, 'benchmark-exec-time' )
+    needs:
+      - building-pr-branch
+      - building-main-branch
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Setup Linux env
+      run: |
+        sudo apt -y update
+        sudo apt install jq podman
+
+    - name: Checkout to PR branch
+      uses: actions/checkout@v2
+
+    - name: Downloading PR build from artifact
+      uses: actions/download-artifact@v2
+      with:
+        name: pr-youki
+        path: ./pr_youki
+
+    - name: Downloading main build from artifact
+      uses: actions/download-artifact@v2
+      with:
+        name: main-youki
+        path: ./main_youki
+
+    - name: Preparing binaries
+      run: |
+        mv ./main_youki/youki ./youki_main
+        mv ./pr_youki/youki ./youki_pr
+        chmod +x ./youki_main
+        chmod +x ./youki_pr
+
+    - name: Preparing benchmark bundle
+      run: |
+        mkdir -p ./benchmark_exec_bundle/rootfs
+        cd benchmark_exec_bundle
+        sudo bash -c 'podman export $(podman create busybox) | tar -C rootfs -xvf -'
+        ../youki_main spec 
+        sed -i 's/"sh"/"sh", "-c", "echo Hi, my PID is $$; echo Bye Bye"/' config.json
+        sed -i 's/"terminal": true/"terminal": false/' config.json
+          
+    - name: Installing hyperfine
+      run: |
+        curl -s https://api.github.com/repos/sharkdp/hyperfine/releases/latest \
+        | jq -r '.assets[] | select(.name? | match("hyperfine-musl_.*_amd64.deb")) | .browser_download_url' \
+        | wget -qi -
+        sudo dpkg -i *.deb
+
+    - name: Running benchmark
+      run: |
+        hyperfine --prepare 'sudo sync; echo 3 | sudo tee /proc/sys/vm/drop_caches' \
+        --export-markdown bench_exec_result.md \
+        --warmup 10 \
+        --min-runs 100 \
+        --command-name 'main youki build' \
+        'sudo ./youki_main run -b benchmark_exec_bundle main_container && sudo ./youki_main delete -f main_container' \
+        --command-name 'pr youki build' \
+        'sudo ./youki_pr run -b benchmark_exec_bundle pr_container && sudo ./youki_pr delete -f pr_container'
+
+    - name: Adding info into result
+      run: |
+        sed -i '1i > commit (${{ github.sha }})\n' bench_exec_result.md
+
+    - name: Writting report to PR comment
+      uses: marocchino/sticky-pull-request-comment@v2 
+      with:
+        append: true
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Maybe this is not enough permissions in the organization, can you change it to print in actions?

Reply:
OK,
I'll figure out a way to print the resulting beauty.

Reply:
@saschagrunert The youki team is trying to measure the results of a very simple benchmark (just for reference) using github actions. I'd like to comment on the results to the PR, but I don't seem to have permission. Could you ask the organizer to check it?

Reply:
@rhatdan can we make @utam0k admin of this repository to add secrets for using GitHub actions?

Reply:
Thanks for your help.
I am already the owner of this repository.
It looks like there is no write on `GITHUB_TOKEN` permissions. Isn't `GITHUB_TOKEN` set to read-only on the organizer side? I could not check the permissions of the organizer with my permissions. In the repository, `GITHUB_TOKEN` is set to write, but it may be possible to override this permission in the organizer.
https://github.com/containers/youki/pull/478/files#diff-28fe3108ee857a8865d400d072ffeab3d06d0c14073313bd0b39cf4fa41a57fdR134-R142